src/warning.rs

```diff
@@ -50,6 +50,17 @@ impl Warning {
                     write(buffer, diagnostic, Severity::Warning);
                     writeln!(buffer, "This code will crash if it is run. Be sure to remove this todo before running your program.").unwrap();
                 }
+                ImplicitlyDiscardedResult { location } => {
+                    let diagnostic = Diagnostic {
+                        title: "Implicitly discarded Result".to_string(),
+                        label: "".to_string(),
+                        file: path.to_str().unwrap().to_string(),
+                        src: src.to_string(),
+                        location: location.clone(),
+                    };
+                    write(buffer, diagnostic, Severity::Warning);
+                    writeln!(buffer, "This code implicitly ignores a Result returned from an expression. Instead, bind the Result to _ if you intend to discard it.").unwrap();
```

Comment:
One of my goals with Gleam is to try and make strong static typing more accessible, so I'd like to try and make the error and warning messaging as newcomer friendly as possible.

How about something more like this:

### Unused result value

The Result value returned by this code is not being used, so any error is being silently ignored.
Check for an error with a case statement, or assign it to the variable `_` if you are sure the error does not matter.

What do you think?

Reply:
I like that! I'm a big fan of Elm-style error messages, and think making static typing more accessible is a noble goal. I was hoping you'd have feedback on the copy :)